rocket/admin.py

```diff
@@ -1,5 +1,12 @@
 from django.contrib import admin
+<<<<<<< HEAD
 from .models import Portal
 # Register your models here.
 
 admin.site.register(Portal)
+=======
```

Comment:
You tried a merge over here and you didn't resolved it. Look for what are merge markers.